2023 sponsor update (#1817)
writethedocs/www: docs/conf/portland/2023/sponsors/prospectus.rst

Sponsorship Packages
====================

> We have broken our sponsorships based on what it is you're trying to get out of the sponsorship.

Reviewer: ```suggestion
We have separated our sponsorships into two streams, each one helping you achieve one of the following specific goals:
```

can be reworded further, but just wanted to move away from 'broken'

> We find that many sponsors have two primary goals in mind:

Reviewer: ```suggestion
```

* Hiring
* Product promotion

Second Draft
------------

> The **Second Draft** package gives you visibility on the primary conference website and communications.

Reviewer: 'primary'? 

```suggestion
The **Second Draft** package gives you visibility on the conference website and communications.
```

It's great for sending folks to the conference,
and showing support for the community.

Benefits
~~~~~~~~

- Three (3) tickets_
- A small logo & short text content (250 characters) on the conference website
- Name included in welcome announcement in email newsletters and social media

The **Second Draft** package costs **{{sponsorship.second_draft.price}}**.

Publisher
---------

> The **Publisher** package gets you a larger online presence and a visible on-site presence, either at the job fair or a sponsored workshop.

Reviewer: This isn't super clear. Are they providing a workshop or sponsoring an existing one.
